src/code/main.c

```diff
@@ -25,21 +25,25 @@ AudioMgr gAudioMgr;
 OSMesgQueue sSerialEventQueue;
 OSMesg sSerialMsgBuf[1];
 
+#ifdef OOT_DEBUG
 void Main_LogSystemHeap(void) {
     PRINTF(VT_FGCOL(GREEN));
     // "System heap size% 08x (% dKB) Start address% 08x"
     PRINTF("システムヒープサイズ %08x(%dKB) 開始アドレス %08x\n", gSystemHeapSize, gSystemHeapSize / 1024, gSystemHeap);
     PRINTF(VT_RST);
 }
+#endif
 
 void Main(void* arg) {
     IrqMgrClient irqClient;
     OSMesgQueue irqMgrMsgQueue;
     OSMesg irqMgrMsgBuf[60];
     uintptr_t systemHeapStart;
     uintptr_t fb;
+#ifdef OOT_DEBUG
     void* debugHeapStart;
     u32 debugHeapSize;
+#endif
```

Comment:
can these temps be moved down into the relevant block about the debug heap? would remove an extra set of ifdefs

Reply:
you may need to make your own block by wrapping the relevant code with {}

Reply:
had to also move another temp down into a scope. but this seems way more likely to be something that was written https://github.com/zeldaret/oot/compare/main...fig02:oot:main-improved?expand=1

I can PR this to you if you want

Reply:
feel free 👍

Reply:
https://github.com/Yanis42/oot/pull/1